CHANGELOG.md

```diff
@@ -0,0 +1,34 @@
+# 3.0
+
+ReactiveCocoa 3.0 includes the first official Swift API, which is intended to
+eventually supplant the Objective-C API entirely.
+
+However, because migration is hard and time-consuming, and because Objective-C
+is still in widespread use, 99% of RAC 2.x code will continue to work under RAC
+3.0 without any changes.
+
+Since the 3.0 changes are entirely additive, this document will discuss how
+concepts from the Objective-C API map to the Swift API. For a complete diff of
+all changes, see [the 3.0 pull
+request](https://github.com/ReactiveCocoa/ReactiveCocoa/pull/1382).
+
+**[Additions](#)**
+
+ 1. [Parameterized types](#)
+ 1. [Interrupted event](#)
+ 1. [Objective-C bridging](#)
+
+**[Replacements](#)**
+
+ 1. [Hot signals are now Signals](#)
+ 1. [Cold signals are now SignalProducers](#)
```

Comment:
I'd say that many people using RAC 2.x are probably not aware of this difference, so it might be worth explaining it beforehand.